client/cypress/support/app.po.ts

```diff
@@ -7,14 +7,6 @@ export class AppPage {
     return cy.get('.app-title');
   }
 
-  getSidenavButton() {
-    return cy.get('.sidenav-button');
-  }
-
-  getSidenav() {
-    return cy.get('.sidenav');
-  }
-
   getNavLink(navOption: 'Pantry' | 'Products') {
```

Comment:
We should have this data type allow for 'Shopping List' as well as pantry and products even if we don't use them in tests currently.

Reply:
That seems like a different issue.

Reply:
I agree that it could be a different issue, but if you want to take that position then you _definitely_ should create the issue. It's definitely weird that this doesn't have `'Shopping List'` and (perhaps more troubling) that there are no tests that seem to care?

Reply:
Actually, @OllieWillette667, I don't think it was a separate issue. I think that my comment below about filling out the tests in `app.spec.ts` would "force" the addition of `'Shopping List'` here. That seems to be the only place that `getNavLink()` is used, and if we make it a little more complete we'll end up addressing @jpwalbran's concern.